CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - The `alces configure` action has gained a `thp` operation for control of transparent hugepages behaviour
 - The Clusterware VPN IP address may optionally be added to DNS and as a SAN within the SSL certificate within a `.vpn` subdomain
+- The `clocksource` option to `alces configure`. Can modify and display the clocksource on a node. Also added a `customization profile` to modify it on `configure`
```

Comment:
Can this be Englishified please? The first two sentences aren't.

"Customization profile" doesn't need to be in backticks. (Actually, not sure that customization profiles need to be mentioned here at all, as we haven't done so in the past - thoughts @mjtko?)